conda-forge.yml

```diff
@@ -1,2 +1,12 @@
 max_py_ver: '37'
 max_r_ver: '35'
+channels:
+    # sources selects the channels to pull packages from, in order.
+    sources:
+      - defaults
+      - intel
+      - conda-forge
+    # targets is a list of 2-lists, where the first element is the
+    # channel to push to and the second element is the label on that channel
+    targets:
+      - ["conda-forge", "main"]
```

Comment:
Please do not edit files outside of the recipe directory

Reply:
@scopatz  thanks. then might be you can help with proper next actions.  as i see only conda-forge target is looked up now. 
But particularly daal-devel dependency is not available in conda-forge, but available in default and intel channels 
For MPI - daal4py supposed to work with intel-mpi that is available in intel channel only. It can be substituted with mpich - but its' not avaialbe for windows. 

How such kind of dependencies can be resolved?  Based on documentation i found only 1 way to handle this through conda-forge.yml

Reply:
Defaults should be avaialble

Reply:
I've end-up with OSX and windows.  so think it worth to go with Linux first and then extend to OSX/Win support